pkg/monitor/operator.go

```diff
@@ -53,6 +78,12 @@ func startClusterOperatorMonitoring(ctx context.Context, m Recorder, client conf
 					if c.Type == configv1.OperatorDegraded && c.Status == configv1.ConditionTrue {
 						level = Error
 					}
+					if c.Type == configv1.OperatorAvailable && c.Status == configv1.ConditionFalse {
+						level = Error
+					}
+					if c.Type == configv1.OperatorProgressing && c.Status == configv1.ConditionTrue {
```

Comment:
I would prefer not to make this an error

Reply:
> I would prefer not to make this an error

Updated to warning for now.